config/settings.go

```diff
@@ -53,6 +53,7 @@ var DefaultSettings = Settings{
 	DefaultPort:               8080,
 	HealthPath:                "/healthz",
 	LogFormat:                 "common",
+	LogLevel:                  "",
```

Comment:
we have to set the default to "info". an empty log level would always produce the error from `main.go:210` if a log level is not set on the startup via CLI:

`logger.Error("parsing log level, falling back to info level")`